sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -892,6 +957,14 @@ export enum HTTPMethod {
     put = "PUT"
 }
 
+// @public (undocumented)
+export interface IEpkRange {
```

Comment:
In .NET and Java we used the term FeedRange and the min/max are not public properties. Since this is exported, would both become public?

Reply:
Yes, this is for passing custom epk ranges.